Doc/library/enum.rst

```diff
@@ -1198,7 +1198,7 @@ constructor.  For example::
     ...     example = '11', 16      # '11' will be interpreted as a hexadecimal
     ...                             # number
     >>> MyEnum.example
```

Comment:
I think that it would more useful to use `MyEnum.example.value` or `int(MyEnum.example)` in the example.

Reply:
Yes, I added the `.value` here and in other places.